src/index.js

```diff
@@ -0,0 +1,69 @@
+/*eslint-disable*/
+import _ from 'lodash';
+/* eslint-enable */
+import './style.css';
+// Create object with different hardcoded elements
+const todolist = [
+  {
+    index: 1,
+    description: 'Wash the dishes',
+    completed: false,
+
+  },
+  {
+    index: 2,
+    description: 'Make my hair',
+    completed: true,
+
+  },
+  {
+    index: 1,
+    description: 'Complete todo list project',
+    completed: false,
+
+  },
+];
+class tasks {
+  constructor() {
+    this.toDoList = [];
+  }
+
+  add(description, completed) {
+    this.toDoList = this.toDoList.concat({ index: Date.now(), description, completed });
+  }
+
+  remove(id) {
+    this.toDoList = this.toDoList.filter((list) => list.id !== Number(id));
+  }
+}
+// Create new tasks in the todolist
+
+/* eslint-disable */
+const lists = new tasks();
+/* eslint-enable */
```

Comment:
- [ ] Kindly remove all `eslint` disable/enable statements in your index.js file. Linters help us write codes in a clean and DRY manner. They also ensure we adopt best practices when writing our code. In this case, you created an instance of the class `tasks` but you have not used it again. Since you are not using the instance of the class, I will advise you to not initialize the class `tasks` nor create an instance of it.